extensions/amp-date-picker/1.0/test/test-component.js

```diff
@@ -0,0 +1,947 @@
+import {expect} from 'chai';
```

Comment:
I don't think we want to use the native chai expect. I think the infra team has build their own wrapper for expect that we can use implicitly.

Reply:
Thanks for catching this, I think my editor pulled it in automatically.